package.json

```diff
@@ -157,12 +173,12 @@
   "scripts": {
     "check": "tslint 'src/**/*.ts' -e src/generated-typings.d.ts",
     "generate-typings": "node server/dev/write-typings.js",
-    "dev": "npm run dev-bundle & npm run dev-start & npm run dev-test",
+    "dev": "concurrently -r 'npm run dev-bundle' 'npm run dev-start' 'npm run dev-test'",
     "dev-build": "npm run dev-bundle",
     "dev-bundle": "npm run generate-typings && jspm bundle -diw 'src/**/*.ts' .tmp/dev-bundle.js --config jspm.dev-bundle.json --log ok",
     "dev-start": "node server/server-dev.js",
     "dev-test": "wait-for-change .tmp/dev-bundle.js && karma start karma-dev.conf.js",
-    "dist": "npm run dist-bundle & npm run dist-test & npm run dist-styles",
+    "dist": "concurrently -r 'npm run dist-bundle' 'npm run dist-test' 'npm run dist-styles'",
```

Comment:
This task fails with the already known
```
TypeError: Path must be a string. Received undefined
      at assertPath (path.js:7:11)
      at Object.dirname (path.js:1324:5)
      at /home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/systemjs-babel-node.js:6645:38
      at /home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/systemjs-babel-node.js:6661:24
      at Array.map (native)
      at OptionManager.resolvePresets (/home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/systemjs-babel-node.js:6650:22)
      at OptionManager.mergePresets (/home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/systemjs-babel-node.js:6640:12)
      at OptionManager.mergeOptions (/home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/systemjs-babel-node.js:6616:16)
      at OptionManager.init (/home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/systemjs-babel-node.js:6732:12)
      at File.initOptions (/home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/systemjs-babel-node.js:7375:77)
      at new File (/home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/systemjs-babel-node.js:7319:24)
      at Pipeline.transform (/home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/systemjs-babel-node.js:3923:18)
      at /home/linne/Dev/ng2-jspm-template/jspm_packages/npm/systemjs-plugin-babel@0.0.9/plugin-babel.js:154:24
  Error loading /home/linne/Dev/ng2-jspm-template/jspm_packages/github/KevCJones/plugin-scss@0.2.11/index.js
```
at least on node 6.2.0 .

Although ... it appears to have nothing to do with this commit, since it fails on master as well.